app/cdap/components/Administration/TetheringTabContent/shared.styles.tsx

```diff
@@ -63,7 +63,9 @@ export const GridHeader = styled.div`
 `;
 
 export const GridBody = styled.div`
-  background-color: var(--white);
+  max-height: 300px;
+  overflow: auto;
+  border-bottom: 1px solid var(--grey11);
```

Comment:
Adding a bottom border to `GridBody` ensures we have the table bottom border even with scrolling where the last row may not necessarily have a full border.